### DISCUSSION TITLE
Understanding poetry build process for wheels

### DISCUSSION THREAD

**Discussion by galleon** [python-poetry/poetry#3755] [Q&A] [answered]
Hi all

I am trying to move my [project](https://github.com/galleon/scikit-decide) to `poetry`. In order to build `wheels` I have created a `build.py`. 
`poetry build` is working just fine (i.e. at least my `whl` is created and looks correct) 
`poetry install` is where I start to have problems with the following error:
```
      [100%] Built target fzn-gecode
      error: can't copy 'build/lib.macosx-10.15-x86_64-3.8/skdecide/hub/.cpython-38-darwin.so': doesn't exist or not a regular file
      ----------------------------------------
  ERROR: Command errored out with exit status 1: /Users/alleon_g/Library/Caches/pypoetry/virtualenvs/scikit-decide-5gp3ZELx-py3.8/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/Users/alleon_g/Documents/0x-TechWork/scikit-decide/tmp/galleon/scikit-decide/setup.py'"'"'; __file__='"'"'/Users/alleon_g/Documents/0x-TechWork/scikit-decide/tmp/galleon/scikit-decide/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' develop --no-deps Check the logs for full command output.
```

Indeed the 

` 'build/lib.macosx-10.15-x86_64-3.8/skdecide/hub/.cpython-38-darwin.so'`  shall be ` 'build/lib.macosx-10.15-x86_64-3.8/skdecide/hub/__skdecide_hub_cpp.cpython-38-darwin.so'`

I have somehow lost the prefix `__skdecide_hub_cpp`

Any idea?

**Comment by abn** [comment]
@galleon is this still an issue?

**Comment by galleon** [comment, marked as answer]
Nope. Thanks I have solved that one.

**Reply by abn** [reply]
Might be good to add what you found here so others coming along can see any common gotchas.
